### PR TITLE
LPS-115881 Provide Audit Graph title

### PR DESCRIPTION
**Pull request by darquesdev**

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:test:sf

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
ci:test:sf

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/ContentDashboardAdminPortlet.java [hunk 1]
```diff
@@ -125,7 +126,11 @@ public void render(
 						_http, _language, liferayPortletRequest,
 						liferayPortletResponse, _portal),
 					_itemSelector, liferayPortletRequest,
-					liferayPortletResponse, _portal, searchContainer);
+					liferayPortletResponse, _portal,
+					ResourceBundleUtil.getBundle(
+						"content.Language", _portal.getLocale(renderRequest),
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/ContentDashboardAdminPortlet.java]: I think this is the same as ResourceBundleUtil.getBundle(
						_portal.getLocale(renderRequest),getClass())

**Comment by darquesdev** [comment]
ci:forward

**Comment by darquesdev** [comment]
ci:forward

**Comment by cgoncas** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**